client/controller/models/apps/apps.go

```
@@ -97,15 +97,12 @@ func Logs(c *client.Client, appID string, lines int) (string, error) {
 
 	body, err := c.BasicRequest("GET", u, nil)
 
-	if err != nil {
-		return fmt.Sprintf("Error:%v", err), err
```

Should we consider outputting `err` instead of swallowing it, in either case?